src/Application/ConfigResolver.php

```diff
@@ -192,13 +192,13 @@ private static function preparePreset(array $preset, array $config): array
         $removedRulesByPreset = [];
         $addedRulesByConfig = [];
 
-        if (isset($preset['remove']) && is_array($preset['remove']) && count($preset['remove']) > 0) {
+        if (isset($preset['remove']) && is_array($preset['remove']) && $preset['remove'] !== []) {
             array_walk_recursive($preset['remove'], static function ($value) use (&$removedRulesByPreset): void {
```

Comment:
Is there any specific reason (benefit) to change this?

Reply:
It is faster

Reply:
Ah! Saw it coming -> https://www.reddit.com/r/PHP/comments/oj1xmk/how_do_you_routinely_check_that_an_array_is_empty/